src/db/build.ts

```diff
@@ -1,9 +1,17 @@
 import { sequelize, Event, User } from '.'
 import fakeData from './FakeData/fakeData.json'
+
+const { NODE_ENV } = process.env
+
 const build = async () => {
   await sequelize.sync({ force: true })
   await Event.bulkCreate(fakeData.Events)
   await User.bulkCreate(fakeData.Users)
 }
 
-build()
+if (NODE_ENV !== 'test') {
+  console.log('success')
+  build()
```

Comment:
it's better to print the message after building the database successfully not before it

and change the message to a more readable one
like this 👇
```suggestion
  build().then(() => console.log('database built successfully !'))
```